src/blenderbim/blenderbim/bim/module/pset/ui.py

```diff
@@ -139,6 +140,11 @@ def draw_psetqto_ui(context, pset_id, pset, props, layout, obj_type):
 def draw_psetqto_editable_ui(box, props, prop):
     row = box.row(align=True)
     draw_property(prop, row, copy_operator="bim.copy_property_to_selection")
+    qto_name = ObjectQtosData.data['qtos'][0]['Name']
+    if qto_name in mapper.keys():
```

Comment:
This if statement should be evaluated in a data class, not in the draw function.

Reply:
Ook, so i have to create a variable (list, dict or something) that reads the mapper dict, right?
But the "if statement" that checks the data remains in the ui.py..is it right?

Reply:
Correct, in the data class or operator import function this should be evaluated as a Boolean. Then the draw function only needs to check a boolean for speed :)